behaviour/graql/language/insert.feature

```diff
@@ -1474,6 +1578,76 @@ Feature: Graql Insert Query
       | BRO |
 
 
+  Scenario: inserting a new instance type on an existing instance is fine if it is already of that type
+    Given graql insert
+      """
+      insert
+      $x isa person, has ref 0;
+      """
+    Given the integrity is validated
+    Then graql insert
+      """
+      match
+        $x isa person;
+      insert
+        $x isa person;
+      """
+    Then the integrity is validated
+    When get answers of graql query
+      """
+      match $x isa person; get;
+      """
+    Then answer size is: 1
+
+
+  Scenario: inserting a new type on an existing instance throws, if the old type is not a subtype of the new one
```

Comment:
"inserting a new type" isn't in our current Grakn vocabulary i think - i would use "re-inserting a matched instance as an unrelated type throws"

the other testss can use similar titles i think using "re-inserting"? What do you think

Reply:
Yes re-insert seems clear enough.

Reply:
Updated to `re-inserting a matched instance as an unrelated type throws an error`